lib/eyeshade/base_balance.rb

```diff
@@ -0,0 +1,51 @@
+module Eyeshade
+  class BaseBalance
+    attr_reader :rates,
+                :default_currency,
+                :amount_probi,
+                :amount_bat,
+                :amount_default_currency,
+                :fees_probi,
+                :fees_bat,
+                :fees_default_currency
+
+    # Account types
+    CHANNEL = "channel".freeze
+    OWNER = "owner".freeze
+
+    def initialize(rates, default_currency)
+      @rates = rates
+      @default_currency = default_currency || "BAT"
+    end
+
+    private
+
+    # Expects and returns a string
+    def probi_to_bat(probi)
+      "#{probi.to_i.to_d / 1E18}"
+    end
+
+    # Expects and returns a string
+    def bat_to_probi(bat)
+      "#{(bat.to_d * BigDecimal.new('1.0e18')).to_i}"
+    end
+
+    # Expects and returns values with probi unit
+    def calc_fees(total_probi)
```

Comment:
can we spell out the entire word here? `calculate_fees` Just makes it less confusing ya' know